.github/workflows/push-foundation-repo.yml

```diff
@@ -1,11 +1,16 @@
-name: Sync Cardinal-Cryptography repo with Aleph-Zero-Foundation repo
+name: Publish AlephBFT main to crates.io.
```

Comment:
You should probably name this something that implies it also synchronizes the foundation repo.

Reply:
Good point, corrected with dc09229.